javascript/javascript3/week1/giphy.js

```diff
@@ -0,0 +1,25 @@
+let search = document.getElementById("search");
+let btn = document.getElementById("btnSearch");
+let divImage = document.getElementById("giphiImage");
+const APIKEY = "L9m45vq8McMRHt49YWmsy1qXVcMzUUK7";
+
+const fetchGiphy = () => {
+  const giphyUrl = `https://api.giphy.com/v1/gifs/search?api_key=${APIKEY}&q=${search.value}`;
+
+  fetch(giphyUrl)
+    .then((response) => response.json())
+    .then((data) => {
+      console.log(data.data[0].images.fixed_width.url);
+      data.data.forEach(function (obj) {
+        console.log(obj.images.fixed_width.url); //log all cats images
```

Comment:
On line number 12 and 14 I can see a lot of gif files is coming from API while search but you are showing only one on the screen. And this part of the homework is also missing "Add an input element, where the user can specify how many gif results the user wants".